api-server/src/main/java/org/opennms/devjam2022/apiserver/model/UserRole.java

```diff
@@ -0,0 +1,14 @@
+package org.opennms.devjam2022.apiserver.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class UserRole {
+    String id;
+    String userIdentity;
```

Comment:
Do we need this userIdentity inside the role?

Reply:
Not really for our current goals. Removed. Thanks